config/openapi.yaml

```diff
@@ -19,8 +19,8 @@ info:
     email: pawel@cierzniakowski.pl
     url: https://cierzniak.ml
 servers:
-- url: http://localhost:8000
-  description: Docker or local development
+  - url: http://localhost:8000
+    description: Docker or local development
```

Comment:
Why did you add these empty spaces? YAML in this project should be intended by 2 spaces and arrays shouldn't be intended 😛